### PR TITLE
Massive KB Docs

### PR DESCRIPTION
**Pull request by mwang87**

### CODE REVIEW

File: docs/massivekb.md [hunk 1]
```diff
@@ -4,7 +4,30 @@ The MassIVE Knowledge Base (MassIVE-KB) is a set of peptide spectral libraries.
 
 ## Browsing Libraries
 
-You can browser MassIVE-KB libraries to explore what is in them, and what is the provenance. 
+You can browser MassIVE-KB libraries to explore what is in them, and what is the provenance. Here is a view of the status page of a massive-kb library:
```
Comment (jjcarver) [inline docs/massivekb.md:7]: 1. "browse", not "browser"
2. "massive-kb" should be capitalized everywhere as "MassIVE-KB".

File: docs/massivekb.md [hunk 2]
```diff
@@ -4,7 +4,30 @@ The MassIVE Knowledge Base (MassIVE-KB) is a set of peptide spectral libraries.
...
+
+![](img/massivekb/kb-status.png)
+
+The following views are described
+
+| View | Description |
```
Comment (jjcarver) [inline docs/massivekb.md:13]: Some descriptions in this table end with a period and some don't.

File: docs/massivekb.md [hunk 3]
```diff
@@ -4,7 +4,30 @@ The MassIVE Knowledge Base (MassIVE-KB) is a set of peptide spectral libraries.
 
 ## Browsing Libraries
 
-You can browser MassIVE-KB libraries to explore what is in them, and what is the provenance. 
+You can browser MassIVE-KB libraries to explore what is in them, and what is the provenance. Here is a view of the status page of a massive-kb library:
+
+![](img/massivekb/kb-status.png)
+
+The following views are described
```
Comment (jjcarver) [inline docs/massivekb.md:11]: This line should probably end with a colon (":").

File: docs/massivekb.md [hunk 4]
```diff
@@ -19,6 +42,66 @@ These spectral libraries can be:
...
+
+### Spectra Extraction/Quality Filtering
+
+After the searches are finished, the identified spectra are extracted from the original mzXML/mzML files and quality filtered. A set of search jobs can be extracted at once using this [workflow](https://proteomics2.ucsd.edu/ProteoSAFe/?params=%7B%22workflow%22:%22LIBRARY_CREATION_SPECTRA_EXTRACTION%22%7D). To select searches to extract, select an appropriate search job from the jobs list. Any jobs run by the specific user may be selected.
+
+
+![](img/massivekb/kb-extraction.png)
+
+
+### Library Creation of a New Library
```
Comment (jjcarver) [inline docs/massivekb.md:71]: The redundant use of "Library" here seems awkward.

File: docs/massivekb.md [hunk 5]
```diff
@@ -19,6 +42,66 @@ These spectral libraries can be:
...
+
+To create a library from an existing spectra extraction workflow, please click the “Create Library from Extracts” link. It will take you to this [workflow](https://proteomics2.ucsd.edu/ProteoSAFe/?params=%7B%22workflow%22:%22LIBRARY_CREATION_AUGMENT_LIBRARY_TEST%22%7D) with selections pre-populated. 
+
+
+![](img/massivekb/kb-create.png)
+
+The default parameters are appropriate for human HCD proteome parameters, adjust accordingly for your data.
+
+### Library Augment of an Existing Library
```
Comment (jjcarver) [inline docs/massivekb.md:80]: The redundant use of "Library" here seems awkward.

File: docs/massivekb.md [hunk 6]
```diff
@@ -19,6 +42,66 @@ These spectral libraries can be:
...
+
+To augment an existing library, on the status page of the existing spectral library, please click the “Augment This Library” link.
+
+![](img/massivekb/kb-augment.png)
+
+This brings users to a library augment workflow page. Please select a spectra extraction job from the jobs table to augment to the existing library. Only one extraction can be augmented to a library at one time.
```
Comment (jjcarver) [inline docs/massivekb.md:86]: "augment" is not a prepositional verb, so the word "to" should be removed immediately after it. Either "augment the library" or "add to the library".

File: docs/massivekb.md [hunk 7]
```diff
@@ -4,7 +4,30 @@ The MassIVE Knowledge Base (MassIVE-KB) is a set of peptide spectral libraries.
```
Comment (jjcarver) [inline docs/massivekb.md:4]: The word "peek" is misspelled as "peak" in the introductory paragraph above. For some reason I could not tag the actual line containing that paragraph with this comment.

**Comment by benpullman** [comment]
Looks good to me too, with Jeremy’s edits.

**Pull request merged**